api/src/main/java/io/opentelemetry/trace/attributes/SemanticAttributes.java

```diff
@@ -127,20 +127,30 @@
    */
   public static final LongAttributeSetter HTTP_RESPONSE_CONTENT_LENGTH_UNCOMPRESSED =
       LongAttributeSetter.create("http.response_content_length_uncompressed");
-  /** The service name, must be equal to the $service part in the span name. */
+
+  /** A string identifying the remoting system, e.g., "grpc", "java_rmi" or "wcf". */
+  public static final StringAttributeSetter RPC_SYSTEM = StringAttributeSetter.create("rpc.system");
+  /** The full name of the service being called, including its package name, if applicable. */
   public static final StringAttributeSetter RPC_SERVICE =
       StringAttributeSetter.create("rpc.service");
-  /** RPC span event attribute with value "SENT" or "RECEIVED". */
-  public static final StringAttributeSetter MESSAGE_TYPE =
+
+  public static final StringAttributeSetter RPC_METHOD = StringAttributeSetter.create("rpc.method");
```

Comment:
I would still add the comment 
```suggestion
/* The name of the method being called, must be equal to the $method part in the span name */
  public static final StringAttributeSetter RPC_METHOD = StringAttributeSetter.create("rpc.method");
```

Reply:
Thanks somehow missed this one

Reply:
Thanks to you for keeping this up2date :)